cypress/e2e/wallet-dropdown.test.ts

```diff
@@ -2,7 +2,7 @@ import { getTestSelector } from '../utils'
 
 describe('Wallet Dropdown', () => {
   beforeEach(() => {
-    cy.visit('/pool')
+    cy.visit('/')
```

Comment:
@vm this is undoing the recent change to run this on /pool because the prior occlusion happening due to landing page updates -- are we sure this is the right change to make?

Reply:
yea we should revert this change